Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -105,6 +105,44 @@ ActsExamples::SeedingAlgorithm::SeedingAlgorithm(
     }
   }
 
+  if (m_cfg.seedFinderConfig.useDetailedDoubleMeasurementInfo) {
+    m_cfg.seedFinderConfig.getTopHalfStripLength.connect([](const void*, const void* ptr) -> float {
+      auto* concretePtr = static_cast<const SimSpacePoint*>(ptr);
+      throw_assert (concretePtr->validDoubleMeasurementDetails() == true, "Not valid measurement details");
```

Comment:
Do you think this is too much caution from our side?

Reply:
throw_assert always does a check, even if compiling without debug flag. This is probably not efficient or necessary, I think "garbage in, garbage out" is fair here.

Reply:
ok, I will remove this check.